src/Mailinglist/Client/MailinglistSubscriptionFailed.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailinglist\Client;
+
+class MailinglistSubscriptionFailed extends \DomainException
```

Comment:
For the front-end, it might be better to work with an `ApiProblem` for consistency?

Reply:
If we add API calls to `udb3-backend` we should indeed try to apply the guides and rules that were decided on.